- topic_name: "clock"
  ros_type_name: "rosgraph_msgs/msg/Clock"
  gz_type_name: "gz.msgs.Clock"
  direction: GZ_TO_ROS

- topic_name: "/bin1/left/camera/points"
  ros_type_name: "sensor_msgs/msg/PointCloud2"
  gz_type_name: "gz.msgs.PointCloudPacked"
  direction: GZ_TO_ROS

- topic_name: "/bin1/right/camera/points"
  ros_type_name: "sensor_msgs/msg/PointCloud2"
  gz_type_name: "gz.msgs.PointCloudPacked"
  direction: GZ_TO_ROS

# - topic_name: "/bin2/left/camera/points"
#   ros_type_name: "sensor_msgs/msg/PointCloud2"
#   gz_type_name: "gz.msgs.PointCloudPacked"
#   direction: GZ_TO_ROS

# - topic_name: "/bin2/right/camera/points"
#   ros_type_name: "sensor_msgs/msg/PointCloud2"
#   gz_type_name: "gz.msgs.PointCloudPacked"
#   direction: GZ_TO_ROS
